### DISCUSSION TITLE
How to change the Docs folder?

### DISCUSSION THREAD

**Discussion by rajtilakjee** [thuliteio/doks#1009] [General]
I have successfully built my site using Doks: https://devmaps.netlify.app/

Next, I would like to restructure my site like this: https://kraken-hpc.io/

1. I want to change the name of the Docs folder
2. I want to add another folder like Docs

I tried changing only the menu from https://getdoks.org/docs/tutorial/add-menus/#global-navigation, however, was unable to build on Netlify. Where and what are the changes I need to make?

Thanks once again.

P.S. I tried to find the Kraken repo but probably it's private. Otherwise, I would have followed the changes from there.

**Comment by rajtilakjee** [comment]
I could do it by myself! Please ignore this.

**Reply by antsanchez** [reply]
How did you do it?

**Reply by rajtilakjee** [reply]
1. Use the following command to create the new folder (remember to rename docs in the command to whatever you want to name the new folder):

npm run create docs

2. Copy all the content of the following folder to the new folder created in the same path:

\devmaps\layouts\docs

**Reply by jundalisay** [reply]
You can try Hextra instead. It lets you arrange by directory while having translations. https://imfing.github.io/hextra/
